customize_image.sh

```diff
@@ -54,11 +56,31 @@ eval set -- "$MYOPTS"
 while true ; do
 	case "$1" in
 		-m|--minimize) MINIMG=1 ; shift ;;
+		-f|--force) FORCE=1 ; shift ;;
 		-h|--help) usage ; exit 0;;
 		*) shift; break ;;
 	esac
 done
 
+SOURCE_IMAGE=$1
+OUTPUT_IMAGE=${2:-$1}
+
+if [ "$SOURCE_IMAGE" == "$OUTPUT_IMAGE" ]; then
+	echo "WARNING: No name for the output image was specified. The source image will be modfied in-place."
+
+	if [ $FORCE -eq 0 ]; then
+		echo ""
+		echo -n "Do you want to continue? [Ny] "
+		read -r choice
+
+		if ! [[ "$choice" == "y" || "$choice" == "y" ]]; then
```

Comment:
Do you mean a capital Y in the "or" part? To check y/Y?

Reply:
Thanks. Fixed in latest push